### PR TITLE
frontend-feat #2055 added shortId, notApplicable ticket and minor UI …

### PR DESCRIPTION
**Pull request by S-PRAVINKUMAR**
…fixes

## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/ui-customer/src/Flow.purs [hunk 1]
```diff
@@ -1629,7 +1648,42 @@ issueReportChatScreenFlow = do
                                                         timestamp : getCurrentUTC ""}) getOptionsRes.messages)
           showSubmitComp = any (\ (Message  message) -> (fromMaybe "" message.label) == "CREATE_TICKET") getOptionsRes.messages 
           isEndFlow' = any (\ (Message  message) -> (fromMaybe "" message.label) == "END_FLOW") getOptionsRes.messages
-      modifyScreenState $ ReportIssueChatScreenStateType (\_ -> updatedState { data {chats = (updatedState.data.chats <> chats'), options = getOptionsRes', chatConfig = updatedState.data.chatConfig{messages = (updatedState.data.chatConfig.messages <> messages')} }, props {isResolved = isResolved, showSubmitComp = (not isResolved && showSubmitComp), isEndFlow = isEndFlow'}})
+      if isEndFlow' then do
```
Comment (aryan-13) [inline Frontend/ui-customer/src/Flow.purs:1651]: use `When`

File: Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/Transformer.purs [hunk 2]
```diff
@@ -112,18 +112,20 @@ isEmailPresent :: LazyCheck -> Boolean
...
-   category : if (getLanguageLocale languageKey == "EN_US")
-                then
-                  joinWith " " (map (\catName ->
-                    let { before, after } = splitAt 1 catName
-                    in (toUpper before <> after)
-                  ) (split (Pattern " ") issue.category))
-                else issue.category,
-   createdAt : getExactTime (runFn2 differenceBetweenTwoUTC (getCurrentUTC "") (issue.createdAt)) issue.createdAt
-}) issueList)
+getApiIssueList issueList = map (\(IssueReportCustomerListItem issue) -> {
```
Comment (aryan-13) [inline Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/Transformer.purs:115]: This can be written as `map mapFunction issueList`

where
  mapFunction ::  --

File: Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/Transformer.purs [hunk 3]
```diff
@@ -112,18 +112,20 @@ isEmailPresent :: LazyCheck -> Boolean
...
+    issueReportId : issue.issueReportId
+  , status : issue.status
+  , category : if getLanguageLocale languageKey == "EN_US" then
```
Comment (aryan-13) [inline Frontend/ui-customer/src/Screens/CustomerUtils/HelpAndSupportScreen/Transformer.purs:118]: This should also be inside helper functions

File: Frontend/ui-customer/src/Flow.purs [hunk 4]
```diff
@@ -1601,7 +1601,26 @@ helpAndSupportScreenFlow = do
...
+              , options = options'
+              , issueId = Just selectedIssue.issueReportId
+              , issueReportShortId = issueInfoRes.issueReportShortId
+              , chatConfig = ReportIssueChatScreenData.initData.data.chatConfig {
+                              messages = messages'
+                            } 
+              }
+          , props {
+              showSubmitComp = false
+            , isResolved = any (\x -> x == selectedIssue.status) ["CLOSED", "NOT_APPLICABLE"]
```
Comment (aryan-13) [inline Frontend/ui-customer/src/Flow.purs:1620]: Better to put this under `let` as well

**Review by aryan-13 — CHANGES_REQUESTED** [review]

File: Frontend/ui-common/src/Components/ChatView/Controller.purs [hunk 5]
```diff
@@ -44,18 +45,8 @@ type Config =
   , showStroke :: Boolean
   , languageKey :: String
   , mapsText :: String
-  , grey700 :: String
```
Comment (kavya-shree-s) [inline Frontend/ui-common/src/Components/ChatView/Controller.purs:47]: @PraveenGongada Why was this added ? 
Can you review this if it is finer removing

Comment (PraveenGongada) [reply Frontend/ui-common/src/Components/ChatView/Controller.purs:47]: These were added because then we don't have common colors.purs file, now they can be removed
